opendp/whitenoise/core/__init__.py

```diff
@@ -0,0 +1,7 @@
+from .base import *
+
+# The full namespace is used because the file location comes from an external repo
+# components.py is generated from rust 
```

Comment:
components.proto is generated from rust. components.py is generated from python.